mat/doc.go

```diff
@@ -42,30 +38,35 @@
 // Receivers must be the correct size for the matrix operations, otherwise the
 // operation will panic. As a special case for convenience, a zero-sized matrix
 // will be modified to have the correct size, allocating data if necessary.
-//  var c mat64.Dense // construct a new zero-sized matrix
+//  var c mat.Dense // construct a new zero-sized matrix
 //  c.Mul(a, a)       // c is automatically adjusted to be 6×6
 //
 // The Matrix Interfaces
 //
-// The Matrix interface is the common link between the concrete types. The Matrix
-// interface is defined by three functions: Dims, which returns the dimensions
-// of the Matrix, At, which returns the element in the specified location, and
-// T for returning a Transpose (discussed later). All of the concrete types can
-// perform these behaviors and so implement the interface. Methods and functions
-// are designed to use this interface, so in particular the method
+// The Matrix interface is the common link between the concrete types of real
+// matrices, The Matrix interface is defined by three functions: Dims, which
+// returns the dimensions of the Matrix, At, which returns the element in the
+// specified location, and T for returning a Transpose (discussed later). All of
+// the concrete types can perform these behaviors and so implement the interface.
+// Methods and functions are designed to use this interface, so in particular the method
 //  func (m *Dense) Mul(a, b Matrix)
 // constructs a *Dense from the result of a multiplication with any Matrix types,
 // not just *Dense. Where more restrictive requirements must be met, there are also the
 // Symmetric and Triangular interfaces. For example, in
 //  func (s *SymDense) AddSym(a, b Symmetric)
 // the Symmetric interface guarantees a symmetric result.
 //
-// Transposes
+// The CMatrix interface plays the same role for complex matrices. The difference
+// is that the CMatrix type has the H method instead T, for returning the Hermitian
+// instead of the raw transpose.
+//
+// Transposes / Hermitians
```

Comment:
Hermitians looks strange here. I would use the full "Transposes and conjugate transposes". We should probably settle either on conjugate or Hermitian in the documentation.